### PR TITLE
First part of Zebra

### PR DESCRIPTION
**Pull request by kevinprince**
So.... Want to pull this to master now so that their is an example of how to do the sessions controller for other applications as well as:

Some basic UI for building everything else inc bootstrap.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,11 +12,10 @@ group :development, :test do
 end
 
 group :assets do
-  gem 'sass-rails',             '3.2.5'
-  gem 'coffee-rails',           '3.2.2'
```
Comment (cbetta) [inline Gemfile:16]: Why are we dropping this?

File: app/assets/javascripts/applications.js [hunk 2]
```diff
@@ -1,10 +0,0 @@
-// This is a manifest file that'll be compiled into including all the files listed below.
-// Add new JavaScript/Coffee code in separate files in this directory and they'll automatically
-// be included in the compiled file accessible from http://example.com/assets/application.js
-// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
-// the compiled file.
-//
-//= require jquery
-//= require jquery_ujs
-//= require twitter/bootstrap
-//= require_tree .
```
Comment (cbetta) [inline app/assets/javascripts/applications.js:10]: Same question

File: app/assets/stylesheets/zebra.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+.form-signin {
+  max-width: 450px;
+  padding: 19px 29px 29px;
+  margin: 0 auto 20px;
+  background-color: #fff;
+  border: 1px solid #e5e5e5;
+  text-align: center;
+}
+
+.form-sign input[type=text]{
```
Comment (cbetta) [inline app/assets/stylesheets/zebra.css]: .form-sign or .form-signin?

File: app/controllers/zebra/sessions_controller.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+    return false if token.empty?
+
+    ticket = token.first.interactable
+
+    unless ticket.kind == Ticket::Kind::CREW
+      return false
+    else
+      return true
+    end
```
Comment (mseckington) [inline app/controllers/zebra/sessions_controller.rb]: The above 5 lines can just be:

```
return ticket.kind == Ticket::Kind::CREW    
```

File: app/controllers/zebra/sessions_controller.rb [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+  def index
+  end
+
+  def show
+    render :text => 'success'
+  end
+
+  def create
+    if check_token(params[:wristband]) === true
```
Comment (cbetta) [inline app/controllers/zebra/sessions_controller.rb]: I'd put this whole behavior in a service object. Something like: 

``` ruby
wristband =  Wristband.find(params[:wristband])
if wristband && wristband.valid?
  ....
end
```

File: app/controllers/zebra/home_controller.rb [hunk 6]
```diff
@@ -0,0 +1,9 @@
+class Zebra::HomeController < Zebra::SessionsController
```
Comment (cbetta) [inline app/controllers/zebra/home_controller.rb]: Bad name,

I still don't get why these need to be 2 different things. What ticket are you solving with this PR?

File: Gemfile (hunk 1 above)
Comment (kevinprince) [inline Gemfile:16]: SASS because bootstrap using less
Coffescript as its evil

File: app/assets/javascripts/applications.js (hunk 2 above)
Comment (kevinprince) [inline app/assets/javascripts/applications.js:10]: Not needed yet

File: app/assets/stylesheets/zebra.css (hunk 3 above)
Comment (kevinprince) [inline app/assets/stylesheets/zebra.css]: Thats a typo, will fix.

**Comment by cbetta** [comment]
I don't get what you're doing in this PR. Please assign it to a ticket and explain what it does.

File: app/controllers/zebra/sessions_controller.rb (hunk 5 above)
Comment (kevinprince) [inline app/controllers/zebra/sessions_controller.rb]: Yep, this code pre-dates the wristband thing i did last night

**Comment by kevinprince** [comment]
I was stubbing out Zebra at OpenHack, this is what it ended up as would rather bring code into master than waste is as it solves the auth piece for Zebra.

**Comment by cbetta** [comment]
Still: what does it do. Who logs in, why, etc. What ticket is this assigned to? #45?

File: app/controllers/zebra/sessions_controller.rb (hunk 4 above)
Comment (cbetta) [inline app/controllers/zebra/sessions_controller.rb]: drop the return even :P

**Comment by kevinprince** [comment]
Closes #45 Allows staff to login with a wristband.

File: spec/controllers/zebra/sessions_spec.rb [hunk 7]
```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+describe Zebra::SessionsController do
```
Comment (mseckington) [inline spec/controllers/zebra/sessions_spec.rb:3]: I'd rewrite this class a bit. I'd structure it with 2 describes: one for #index and one for #new. Then #new would have 3 contexts: without a valid token, with a valid token and with an invalid token.

**Comment by kevinprince** [comment]
Ok lets try this again....

**Comment by kevinprince** [comment]
:ok: by me and @mseckington at Shoreditch Grind.

**Pull request merged**